Makefile

```diff
@@ -16,6 +16,10 @@ api-lint: ## Lint the backend
 api-lint-fix: ## Lint and fix the API
 	@docker compose run --rm api sh -c "black . && isort ."
 
+.PHONY: api-shell
+api-shell: ## Open django shell
+	@docker-compose exec api sh -c "./manage.py shell"
```

Comment:
```suggestion
	@docker-compose exec api ./manage.py shell
```

Reply:
done here: https://github.com/adfinis/Outdated/pull/86/commits/c2f24526c07329b0de009867b1a3ea751541062a

Reply:
Not sure if this is a GH issue but I don't see any change to the previous state..

Reply:
sorry, I forgot to push it.
done here: https://github.com/adfinis/Outdated/pull/86/commits/e8c395633e6fce1f87e4d8c988dfc9d11961bc22

Reply:
Better :+1: